scripts/DSRA_outputs2postgres_lfs.py

```diff
@@ -146,11 +146,16 @@ def GetDataframeForScenario(url, repo_list, retrofitPrefix, eqscenario, columnCo
     #dfLosses['sl_BldgT'] = dfLosses['sl_Str'] + dfLosses['sl_NStr'] + dfLosses['sl_Cont']  
     dfLosses = dfLosses.add_suffix("_{}".format(retrofitPrefix))
     dfLosses.rename(columns={"AssetID_{}".format(retrofitPrefix):"AssetID"}, inplace=True)
+
+    eq_url = f'https://raw.githubusercontent.com/OpenDRR/earthquake-scenarios/master/initializations/s_Hazard_{eqscenario}.ini'
```

Comment:
Nice use of f-strings. We have been using "string".format() quite a bit through the code base but this works too